e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/UpdateNodeE2eTest.java

```diff
@@ -131,6 +132,14 @@ public static void beforeAll()
         WireMock.configureFor(hxInsightMock.getHost(), hxInsightMock.getPort());
     }
 
+    @AfterEach
+    public void reset()
```

Comment:
Maybe method name could be more descriptive like: `resetWiremock()`?

Reply:
I'm not sure if that method is even needed - I've added it recently as I thought it'd fix the failing E2E test - but it doesn't seem to help. What is worse - on my local machine, I can't get the same test to fail...

Reply:
I think I've found a descent way to fix that failing test and will use that method, so I'll rename it.